lib/cmd/fh3/admin/environments/create.js

```diff
@@ -16,9 +16,8 @@ module.exports = {
       _id : params.id, 
       label : params.label, 
       targets : targets, 
-      autoDeployOnCreate : params.autoDeployOnCreate,
-      autoDeployOnUpdate : params.autoDeployOnUpdate
+      autoDeployOnCreate : params.autoDeployOnCreate || false,
+      autoDeployOnUpdate : params.autoDeployOnUpdate || false
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1125)